srcpkgs/guile/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'guile'
 pkgname=guile
 version=2.0.14
-revision=5
+revision=6
 build_style=gnu-configure
 configure_args="--disable-static --disable-error-on-warning"
 hostmakedepends="pkg-config"
```

Comment:
While you're at it can you fix license= ?

Reply:
Done. :)

Reply:
thank you